samples/bluetooth/ibeacon/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _bluetooth-ibeacon-sample:
+.. zephyr:code-sample:: bluetooth_ibeacon
+   :name: iBeacon
+   :relevant-api: bluetooth
 
-Bluetooth: iBeacon
-##################
+   Advertise an Apple iBeacon using BLE Broadcaster role.
```

Comment:
```suggestion
   Advertise an Apple iBeacon using GAP Broadcaster role.
```

The Broadcaster role is defined by the Generic Access Profile (GAP)

Reply:
i took this from the overview text :) i'll fix it too, i guess
<img width="825" alt="image" src="https://github.com/user-attachments/assets/fdc70fbf-2cd8-4f04-a3fc-7b98aeb1d2c1">

Reply:
That makes sense. You can leave it as is for now and then we can follow-up

Reply:
marking as resolved then